src/JsonSchema/Constraints/ConstraintInterface.php

```diff
@@ -59,5 +59,10 @@ public function isValid();
      * @param mixed            $i
      * @throws \JsonSchema\Exception\ExceptionInterface
      */
+    public function _check(&$value, $schema = null, JsonPointer $path = null, $i = null);
```

Comment:
@bighappyface Is the only active contributor with write privileges. He's usually pretty mellow, but no harm in getting his opinion now.